pom.xml

```diff
@@ -23,7 +23,7 @@
     <groupId>org.wso2.carbon.connector</groupId>
     <artifactId>org.wso2.carbon.connector.SMPP</artifactId>
     <packaging>jar</packaging>
-    <version>1.0.7</version>
+    <version>1.0.8</version>
```

Comment:
Since we are upgrading the connector model and the SDK, it's better to do a major release like `1.1.0`. @arunans23 WDYT?

Reply:
Yes. +1

Reply:
I have updated the connector version to 1.1.0 with https://github.com/wso2-extensions/esb-connector-smpp/pull/27/commits/7e5094daa75e0da673cb023c503d8dd37120aedf